drivers/gles3/storage/config.cpp

```diff
@@ -34,6 +34,15 @@
 #include "core/config/project_settings.h"
 #include "core/templates/vector.h"
 
+#if !defined(GLES_OVER_GL) && !defined(WEB_ENABLED) && !defined(IOS_ENABLED)
```

Comment:
Should this just be #ifdef ANDROID_ENABLED or is there another platform define that should be included?

Reply:
That's a good point! As far as I know, EGL is just an Android thing. I'll update these